install/roles/qpc_server/tasks/main.yml

```diff
@@ -20,26 +20,24 @@
     server_port: "{{ ansible_env['SERVER_PORT'] | default('443') }}"
   when: server_port is not defined
 
-- name: Set server url
+- name: set default server url
```

Comment:
For consistency, can you capitalize the first word of each task name? ie. change `set .. ` to `Set .. ` for this line and lines 32 & 37.